modules/dkan_js_frontend/tests/src/Functional/Controller/DatasetControllerTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace modules\dkan_js_frontend\tests\Functional\Controller;
+
+use Drupal\Core\Url;
+use Drupal\Tests\BrowserTestBase;
+use GuzzleHttp\RequestOptions;
+
+class DatasetControllerTest extends BrowserTestBase {
```

Comment:
This isn't actually testing the dkan_js_frontend controller, because the module is not enabled for this test. It's getting a 404 because it's not a valid URL either way, but it's just coming from normal Drupal routing and node controller. That's why it's getting flagged for test coverage.

Reply:
Thanks @dafeder. That hadn't occurred to me. I'll fix.